<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWnd</class>
 <widget class="QDialog" name="CMainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BB Notifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CallNotifierServer.qrc">
    <normaloff>:/Images/GreenIco.ico</normaloff>:/Images/GreenIco.ico</iconset>
  </property>
  <widget class="QListWidget" name="listHistory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>263</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearHistory">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Clear History</string>
   </property>
  </widget>
  <widget class="QLinkLabel" name="linkLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>275</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Donate to future project development...
We have a lot of great ideas and features to implement !!!</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: 75 italic;
	text-decoration: underline;
}
</string>
   </property>
   <property name="text">
    <string>Donate &amp; advice...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLinkLabel</class>
   <extends>QLabel</extends>
   <header>LinkLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CallNotifierServer.qrc"/>
 </resources>
 <connections/>
</ui>
